learning-resources/index.md

```diff
@@ -45,3 +47,7 @@ Feel free to add and remove headings as you wish and think it fits the content b
 
 ## MISC
 ```
+
+### Editor Used
```

Comment:
Why is this useful for anyone to know about it?

Reply:
Most newcomers are used to Windows and don't really understand a markdown format. They usually work with .txt files with the help of Windows format, so markdown initially for newcomers seems like pretty complicated code.
Visual Studio Code has a preview option built in ( I think ) that enables you render markdown files as you make changes to them.
It might help beginners to start using a new tool and learn a new textual format.

Reply:
alright 👍

Reply:
@Rubix982 I was thinking we can move it to making contributions page. since this info will be the same for all the sections, it doesn't belong to learning resources. What do you think?

Reply:
Yes, it does make sense. Sure, will work on it. (Y) 

CC, @azmaktr